drivers/sensor/shtcx/shtcx.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2021 Thomas Stranger
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SHTCX_SHTCX_H_
+#define ZEPHYR_DRIVERS_SENSOR_SHTCX_SHTCX_H_
+
+#include <device.h>
+#include <devicetree.h>
+#include <kernel.h>
+#include <drivers/gpio.h>
+
+/* common cmds */
+#define SHTCX_CMD_READ_ID              0xEFC8
+#define SHTCX_CMD_SOFT_RESET           0x805D
+/* shtc3 only: */
+#define SHTCX_CMD_SLEEP                0xB098
+#define SHTCX_CMD_WAKEUP               0x3517
+
+#define SHTCX_POWER_UP_TIME_US   240U
+/* Soft reset time is 230us for shtc1 and 240us for shtc3 */
+#define SHTCX_SOFT_RESET_TIME_US 240U
+
+#define SHTCX_MAX_READ_LEN 6
```

Comment:
Please indent/align it whit tabs right away.

Reply:
Done.